step-templates/roundhouse-database-migration.json

```diff
@@ -0,0 +1,257 @@
+        "DefaultValue": "",
+        "DisplaySettings": {
+          "Octopus.ControlType": "SingleLineText"
+        }
+      },
+      {
+        "Id": "66e7ce09-798a-42a6-9e4e-f4c952789af8",
+        "Name": "roundhouseViewsFolder",
+        "Label": "ViewsFolder Path",
+        "HelpText": "Filepath for view scripts.  If left blank, .\\views is used.",
+        "DefaultValue": "",
+        "DisplaySettings": {
+          "Octopus.ControlType": "SingleLineText"
+        }
+      },
+      {
+        "Id": "5d905d17-c041-4981-9521-7a827bac3a87",
+        "Name": "roundhouseSprocsFolder",
+        "Label": "SprocsFolder Path",
+        "HelpText": "File path for stored procedures.  If left blank, .\\sprocs is used.",
+        "DefaultValue": "",
+        "DisplaySettings": {
+          "Octopus.ControlType": "SingleLineText"
+        }
+      },
+      {
+        "Id": "fa34cd00-3479-47cd-94d1-637263ebab71",
+        "Name": "roundhouseIndexFolder",
+        "Label": "IndexFolder Path",
+        "HelpText": "File path for Index scripts.  If left blank, .\\indexes is used.",
+        "DefaultValue": "",
+        "DisplaySettings": {
+          "Octopus.ControlType": "SingleLineText"
+        }
+      },
+      {
+        "Id": "e97ba5b9-da0d-4395-887f-f49dcce521f3",
+        "Name": "roundhouseRunAfterAnyTimeFolder",
+        "Label": "RunAfterAnytimeFolder Path",
+        "HelpText": "File path for running scripts after the Other Anytime scripts.  If left blank, .\\runAfterOtherAnyTimeScripts is used.",
+        "DefaultValue": "",
+        "DisplaySettings": {
+          "Octopus.ControlType": "SingleLineText"
+        }
+      },
+      {
+        "Id": "ac179614-fa54-4a5c-a736-7a0cbaa7e4bf",
+        "Name": "roundhousePermissionsFolder",
+        "Label": "PermissionsFolder path",
+        "HelpText": "Folder path that holds permission scripts.  If left blank, .\\permissions is used.",
+        "DefaultValue": "",
+        "DisplaySettings": {
+          "Octopus.ControlType": "SingleLineText"
+        }
+      },
+      {
+        "Id": "fcf2e6c4-db73-4b44-bf36-44e7b1cdc8c1",
+        "Name": "roundhouseDownloadNuget",
+        "Label": "Download RoundhousE?",
+        "HelpText": "Check this box if you want the template to download RoundhousE and use the downloaded version for deployment.  Requires .NET Core be installed on the machine executing the deployment.",
+        "DefaultValue": "False",
+        "DisplaySettings": {
+          "Octopus.ControlType": "Checkbox"
+        }
+      },
+      {
+        "Id": "62b65afa-1a88-4013-83ab-0b23e162d3cf",
+        "Name": "roundhouseNugetVersion",
+        "Label": "Version of RoundhousE",
+        "HelpText": "Version of RoundhousE to download (used with Download RoundhousE option)",
+        "DefaultValue": "",
+        "DisplaySettings": {
+          "Octopus.ControlType": "SingleLineText"
+        }
+      },
+      {
```

Comment:
Can we make this the first parameter? That will match our built-in steps closer.

Reply:
Done.